apps/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -21,10 +21,12 @@ import { LastLinkFlag, LastLinkFlagFactory } from '../../api/LastLinkFlag';
 import { RushConfigurationProject } from '../../api/RushConfigurationProject';
 import { BuildCacheConfiguration } from '../../api/BuildCacheConfiguration';
 import { SelectionParameterSet } from '../SelectionParameterSet';
-import { CommandLineConfiguration, IPhase, IPhasedCommand } from '../../api/CommandLineConfiguration';
-import { IProjectTaskSelectorOptions, ProjectTaskSelector } from '../../logic/ProjectTaskSelector';
-import { IFlagParameterJson } from '../../api/CommandLineJson';
+import type { CommandLineConfiguration, IPhase, IPhasedCommand } from '../../api/CommandLineConfiguration';
+import type { IProjectTaskSelectorOptions } from '../../logic/ProjectTaskSelector';
+import { ProjectTaskSelector } from '../../logic/ProjectTaskSelector';
```

Comment:
Does our TS version support 

```suggestion
import { ProjectTaskSelector , type IProjectTaskSelectorOptions } from '../../logic/ProjectTaskSelector';
```

yet?

Reply:
Needs 4.5+